src/System.Text.Json/src/System/Text/Json/Node/JsonArray.cs

```diff
@@ -0,0 +1,668 @@
+            }
+        }
+
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="JsonArray"/> class representing the specified collection of <see cref="sbyte"/>s.
+        /// </summary>
+        /// <param name="values">Collection to represent.</param>
+        [CLSCompliant(false)]
+        public JsonArray(IEnumerable<sbyte> values) : this()
+        {
+            foreach (sbyte value in values)
+            {
+                _list.Add(new JsonNumber(value));
+            }
+        }
+
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="JsonArray"/> class representing the specified collection of <see cref="ushort"/>s.
+        /// </summary>
+        /// <param name="values">Collection to represent.</param>
+        [CLSCompliant(false)]
+        public JsonArray(IEnumerable<ushort> values) : this()
+        {
+            foreach (ushort value in values)
+            {
+                _list.Add(new JsonNumber(value));
+            }
+        }
+
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="JsonArray"/> class representing the specified collection of <see cref="uint"/>s.
+        /// </summary>
+        /// <param name="values">Collection to represent.</param>
+        [CLSCompliant(false)]
+        public JsonArray(IEnumerable<uint> values) : this()
+        {
+            foreach (uint value in values)
+            {
+                _list.Add(new JsonNumber(value));
+            }
+        }
+
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="JsonArray"/> class representing the specified collection of <see cref="ulong"/>s.
+        /// </summary>
+        /// <param name="values">Collection to represent.</param>
+        [CLSCompliant(false)]
+        public JsonArray(IEnumerable<ulong> values) : this()
+        {
+            foreach (ulong value in values)
+            {
+                _list.Add(new JsonNumber(value));
+            }
+        }
+
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="JsonArray"/> class representing the specified collection of <see cref="decimal"/>s.
+        /// </summary>
+        /// <param name="values">Collection to represent.</param>
+        public JsonArray(IEnumerable<decimal> values) : this()
+        {
+            foreach (decimal value in values)
+            {
+                _list.Add(new JsonNumber(value));
+            }
+        }
+
+        /// <summary>
+        ///   Gets or sets the element at the specified index.
+        /// </summary>
+        /// <param name="idx">The zero-based index of the element to get or set.</param>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///   Index is less than 0.
+        /// </exception>
+        public JsonNode this[int idx]
+        {
+            get => 0 <= idx && idx < _list.Count ? _list[idx] : throw new ArgumentOutOfRangeException();
```

Comment:
>0 [](start = 19, length = 1)

NIT: can we add parenthesis around the condition? having a `=> 0 <=` is not super readable.

Reply:
Or just make it a statement-bodied member and write the braces, if, and return.  (Much more legible, IMO)

Reply:
Right, sure, I'll change it.